intelmq/bots/parsers/shadowserver/README.md

```diff
@@ -24,7 +24,9 @@ Possible feednames:
 * `Accessible-ADB`
 * `Accessible-Cisco-Smart-Install`
 * `Accessible-CWMP`
+* `Accessible-DB2`
```

Comment:
This report is about open DB2 discovery services. So we should probably call it "Open-DB2-Discovery-Service" like "Open-NetBIOS-Nameservice"?

Reply:
I'm open to the change. It'll be based on shadowserver description.